typings/index.d.ts

```diff
@@ -11,8 +11,8 @@ interface Queries {
 }
 
 export type RenderResult<Q extends Queries = typeof queries> = {
-  container: HTMLElement
-  baseElement: HTMLElement
+  container: HTMLDivElement | HTMLElement
+  baseElement: HTMLBodyElement | HTMLDivElement | HTMLElement
```

Comment:
I don't think this'll do what you want it to do. You'll still get the warning because `.firstChild` isn't on _all_ the possible types for the `container` or `baseElement`.

I think the real solution here is to remove `HTMLElement` in addition to your changes. Could you try that out by modifying this file in your `node_modules` first and say what happens?

Reply:
Yeah!

Reply:
Corrected!